src/app/backend/logs.go

```diff
@@ -0,0 +1,101 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"io/ioutil"
+	api "k8s.io/kubernetes/pkg/api"
+	client "k8s.io/kubernetes/pkg/client/unversioned"
+	"strconv"
+	"time"
+)
+
+// Log request structure
+type LogRequest struct {
+	// Pod id requested for logs
+	PodId string `json:"podId"`
+	// Pod namespace
+	PodNamespace string `json:"podNamespace"`
+}
+
+// Log response structure
+type Logs struct {
+	// Pod id
+	PodId string `json:"podId"`
+	// Specific date (RFC3339) when logs started
+	SinceTime string `json:"sinceTime"`
+	// Logs string
+	Logs string `json:"logs"`
+}
+
+// Return logs for particular pod or error when occurred.
+func LogApp(logRequest *LogRequest, client *client.Client) (*Logs, error) {
```

Comment:
"LogApp"? I don't know what this name means. How about something that says "get pod logs"?